velox/functions/lib/window/RowNumber.cpp

```diff
@@ -52,24 +65,35 @@ class RowNumberFunction : public exec::WindowFunction {
 
 } // namespace
 
-// Signature of this function is : row_number() -> bigint.
-void registerRowNumber(const std::string& name) {
+void registerRowNumber(
+    const std::string& name,
+    const TypeKind& offsetTypeKind) {
```

Comment:
Nit : rename "resultTypeKind"

const& not needed with TypeKind enum.

Reply:
Updated.